client-data/js/board.js

```diff
@@ -67,7 +67,7 @@ Tools.connect = function () {
 	var token = params.get("token");
 
 	this.socket = io.connect('', {
-		"query": "token=" + token,
+		...(token && {"query" : "token=" + token}),
```

Comment:
Unfortunately, we can't use "modern" js in wbo, which supports old browsers and doesn't include a js compilation step

Reply:
Okay, I'll put an if block around the whole lot to deal with this instead

Reply:
I would do:

```js
        var socket_params = {
            "x": "y"
        };
        if (params.has("token")) socket_params.token = params.get("token");
	this.socket = io.connect('', socket_params)
```

Reply:
👍 Much better, will do